src/main/java/nextstep/ladder/model/Lines.java

```diff
@@ -0,0 +1,32 @@
+package nextstep.ladder.model;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Lines {
+    private List<Line> lines;
+
+    public Lines(List<Line> lines) {
+        this.lines = lines;
+    }
+
+    public int doResult(int start) {
+        int x = start;
+        int y = 0;
+        int height = lines.get(0).getPoints().size();
```

Comment:
```suggestion
        int height = lines.get(0)
            .getPoints()
            .size();
```
이 [코멘트](https://github.com/next-step/java-ladder/pull/1112#discussion_r702291739)에 대한 답변을 드리면
제시한 코드와 같습니다.

단순한 줄바꿈이지만 연속적으로 일을 처리하는 행위를 한 줄, 한 줄로 만들어 코드를 이해하기 쉽게 만들어줍니다.